.vale/styles/RedHat/PassiveVoice.yml

```diff
@@ -64,11 +64,15 @@ tokens:
   - fought
   - found
   - frozen
+  - get
```

Comment:
so these need to be part of the regex, not on their own. If we put them here on their own they will flag way too many things. The regex is    \b(am|are|were|being|is|been|was|be)\b\s*

Reply:
Done! Updated as suggested!